docs/admin/oauth2/create-client.md

```diff
@@ -0,0 +1,32 @@
+# Register Client in Hopsworks
+
+Before registering a client in Hopsworks you need to create a client application in your identity provider and 
+acquire a _client id_ and _client secret_. An example on how to create a client using [Okta](https://www.okta.com/) 
+identity provider can be found [here](./create-okta-client.md).
+
+After acquiring the _client id_ and _client secret_ create the client in Hopsworks by giving it a name (the name 
+will be used in the login page) as shown in the figure below.
```

Comment:
Relying on the people using the image is not good from an accessibility point of view. People with bad view can't use it.
On the picture I can see 3 fields that are not described in this page:
- Connection URL
- Verify email
- Code challenge.

We need to have an explanation for each of these in this page.